src/common/constants.ts

```diff
@@ -41,6 +41,7 @@ const Constants = {
     },
     Staff: {
       ModChat: 'mod_chat',
+      AdminChat: 'admin_chat',
```

Comment:
not necessary https://github.com/joey-colon/lion/blob/5d686fd2ead1da4121b2e2348e5401c5da0d683d/src/app/plugins/addclasschans.plugin.ts#L15